packages/api/test/e2e/queries.e2e.ts

```diff
@@ -116,7 +116,8 @@ describe('e2e queries', () => {
   describe('Staking account derived query', () => {
     it('Gets staking account details', async done => {
       const stashId = aliceStash.address;
-      const stakingAccount = await api.derive.staking.accountInfo(stashId);
+      const currentEra = await api.query.staking.currentEra();
```

Comment:
should be activeEra can be null only for a few blocks at genesis until era 0 starts
```suggestion
      const activeEra = await api.query.staking.activeEra().unwrap().index;
```